README.md

```diff
@@ -28,6 +28,8 @@ kind: DNSRecordSet
 metadata:
   name: <zone-name>
   namespace: alpha-dns
+  labels:
+    codeRepo: <GitHub-repository>
```

Comment:
IDK if linking to another github repo is where we want to start this. What if we started something like 
```suggestion
    ownerName: John Doe
    ownerContact: John.Doe@google.com 
```

Reply:
I suppose PHAC contact info is public already.

Reply:
Ack - should have used the "add suggestion to batch" - can you reopen that? (Added more commits to branch instead - open to suggestions! )